content/en/blog/2022/how-to-get-started-with-opentelemetry-go.md

```diff
@@ -0,0 +1,420 @@
+---
+title: Go Web App Instrumentation
```

Comment:
Rename the file to more closely match the title, so something like: `go-web-app-instrumentation.md`.